mne/preprocessing/xdawn.py

```diff
@@ -314,23 +383,20 @@ def transform(self, epochs):
             Spatially filtered signals.
         """
         if isinstance(epochs, _BaseEpochs):
-            data = epochs.get_data()
+            epochs_data = epochs.get_data()
         elif isinstance(epochs, np.ndarray):
-            data = epochs
+            epochs_data = epochs
         else:
             raise ValueError('Data input must be of Epoch '
                              'type or numpy array')
 
-        # create full matrix of spatial filter
-        full_filters = list()
-        for filt in self.filters_.values():
-            full_filters.append(filt[:, 0:self.n_components])
-        full_filters = np.concatenate(full_filters, axis=1)
+        return self._transform_xdawn(epochs_data)
 
-        # Apply spatial filters
-        X = np.dot(full_filters.T, data)
-        X = X.transpose((1, 0, 2))
-        return X
+    @property
+    def get_evoked(self, cond):
+        """EvokedArray of specified condition."""
+        return EvokedArray(self.evoked_[cond], info=self._info,
+                           tmin=self._tmin)
```

Comment:
is this an enhancement or is it really necessary? If the latter, keep, else remove